packages/shared/src/components/post/PostItemCard.tsx

```diff
@@ -41,66 +41,79 @@ export default function PostItemCard({
     onHide({ postId: post.id, timestamp: timestampDb });
   };
   const article = post?.sharedPost ?? post;
+  const isUnknownSource = post?.sharedPost?.source?.id === 'unknown';
```

Comment:
Is this needed on reading history?
If so you also need to change the href, as for those cases commentsPermalink is not a external link.

But please double check which case we need to go external from reading history